src/snowflake/telemetry/_internal/opentelemetry/exporter/otlp/proto/common/metrics_encoder.py

```diff
@@ -0,0 +1,11 @@
+from opentelemetry.exporter.otlp.proto.grpc.metric_exporter import OTLPMetricExporter
+from opentelemetry.proto.collector.metrics.v1.metrics_service_pb2 import ExportMetricsServiceRequest
+from opentelemetry.sdk.metrics.export import MetricsData
+
+
+def encode_metrics(data: MetricsData) -> ExportMetricsServiceRequest:
+    exporter = OTLPMetricExporter()
```

Comment:
Can we avoid creating a new instance every time we call this and maybe have an internal global instance that we call the `_translate_data` on?

Reply:
Fixed in latest commit